doc/nrf/includes/nRF54H20_erase_UICR.txt

```diff
@@ -2,11 +2,16 @@
    Programming the nRF54H20 SoC can sometimes fail due to conflicts in the resource configuration.
    This can happen if, for example, an application programmed to the nRF54H20 SoC configured the UICRs for one or more cores in a way that is incompatible with the configuration required by the application you are trying to program on the SoC.
 
-   To fix this error and erase the UICRs, run the following commands::
+   To fix this error and erase the UICRs for the application core, run the following command::
```

Comment:
```suggestion
   To fix this error and erase the UICR for the application core, run the following command::
```
I'm not sure what the convention is here, but `UICRs` in plural sounds weird to me, if it only refers to the UICR of the application core.